game/src/engine/audio/AudioFile.h

```diff
@@ -7,8 +7,15 @@
 struct AudioFile
 {
     int number_of_channels_;
-    int samples_per_channel;
-    int sample_rate_;
+    ALuint samples_per_channel;
+    ALsizei sample_rate_;
     ALenum format_;
-    std::unique_ptr<short> data_;
+    std::unique_ptr<ALshort> data_;
+
+    ALsizei get_size_bytes()
```

Comment:
iirc the actual Google Style Guide does say to do getters/setters in snake_case instead of UpperCamel, but I think all other getters/setters in our code are currently UpperCamel, so maybe rename to `GetSizeBytes()` for consistency?

Reply:
whoops! force of habit, usually i catch these before pushing !